### ISSUE TITLE
[BUG] rule "<fieldset> element does not have a <legend>"

### ISSUE THREAD

**Issue by masi** [IBMa/equal-access#265]
The browser plugin in version 3.1.0 using WCAG 2.1 (A, AA) from 7th of October complains about a missing legend when the fieldset has one.

This is the code (created by a form rendering lib):

```
<fieldset class="powermail_fieldset powermail_fieldset_85 ">
	<legend class="powermail_legend">Anmeldung Führung Studieninformation</legend>

<div class="powermail_fieldwrap powermail_fieldwrap_type_input powermail_fieldwrap_schuleorganisation  form-group wpMovingFormGroup">
    <label for="powermail_field_schuleorganisation" class="" title="">
        <input required="required" data-parsley-required-message="Dieses Feld muss ausgefüllt werden!" data-parsley-trigger="change" class="powermail_input form-control" id="powermail_field_schuleorganisation" type="text" name="tx_powermail_pi1[field][schuleorganisation]" value="">
        <span class="wpMovingLabel">Schule/Organisation<span class="mandatory">*</span></span>
    </label>
</div>

<div class="powermail_fieldwrap powermail_fieldwrap_type_input powermail_fieldwrap_alterteilnehmer_innen  form-group wpMovingFormGroup">
    <label for="powermail_field_alterteilnehmer_innen" class="" title="">
        <input required="required" data-parsley-required-message="Dieses Feld muss ausgefüllt werden!" data-parsley-trigger="change" class="powermail_input form-control" id="powermail_field_alterteilnehmer_innen" type="text" name="tx_powermail_pi1[field][alterteilnehmer_innen]" value="">
        <span class="wpMovingLabel">Alter Teilnehmer_innen<span class="mandatory">*</span></span>
    </label>
</div>

<!--- more fields removed for brevity -->	

</fieldset>
```

**Comment by tombrunet** [comment]
Sorry about the delay, but I am unable to reproduce this with the latest ruleset and the provided HTML...

**Comment by masi** [comment]
Yes, no errors now. Thank you.
